app/controllers/blog_posts_controller.rb

```diff
@@ -24,4 +25,12 @@ def show
   def pro_blog_posts
     Pro::BlogPost.includes(:network, :persons)
   end
+
+  def themed_blog_posts
```

Comment:
@lbiedinger I take your point, but what is not covered by the controller yet is the fact that in the index action we will never be requesting _all_ blog posts from Pro, but will always filter them by some tag (or other Bolt taxonomy term). I'll leave this as-is for the moment and revisit when we have full, categorised, test data to work against.